src/gui/transferlistmodel.cpp

```diff
@@ -58,7 +62,27 @@ static bool isDarkTheme();
 // TransferListModel
 
 TransferListModel::TransferListModel(QObject *parent)
-    : QAbstractListModel(parent)
+    : QAbstractListModel {parent}
+    , m_statusStrings {
+          {BitTorrent::TorrentState::Downloading, tr("Downloading")},
```

Comment:
Can you test if `lupdate` picks up the strings from this construct?
If yes, just go ahead with the merge.